src/js/components/TabPanesComponent.jsx

```diff
@@ -33,6 +33,36 @@ var TabPanesComponent = React.createClass({
     this.updateCurrentGroup();
   },
 
+  getContextualBar: function () {
+    var state = this.state;
+
+    if (state.filters.filterText == null ||
+        Util.isEmptyString(state.filters.filterText)) {
+      return <BreadcrumbComponent groupId={state.currentGroup} />;
+    }
+
+    return (
+        <p className="breadcrumb">
+          <span>{`Search results for "${state.filters.filterText}"`}</span>
+          <a href="#"
```

Comment:
Could you use ```<Link>``` instead of ```<a>``` like here https://github.com/mesosphere/marathon-ui/blob/master/src/js/components/SidebarComponent.jsx#L40 ?